File: insercion al final.cpp
#include <iostream>
using namespace std;

int main()
{
    int inputArr[] = {11, 21, 31, 41, 51, 61};
    int n = sizeof(inputArr) / sizeof(inputArr[0]);
    int ele = 52;

    int nuevoArr[7];

    cout << "Antes de la inserción, el array es: " << endl;
    for (int j = 0; j < n; j++)
    {
        cout << inputArr[j] << " ";
    }

    for (int j = 0; j < n; j++)
    {
        nuevoArr[j] = inputArr[j];
    }
    nuevoArr[n] = ele;

    cout << "\nDespués de la inserción, el array es: " << endl;
    for (int j = 0; j < n + 1; j++)
    {
        cout << nuevoArr[j] << " ";
    }
    cout << endl;

    return 0;
}

File: Arreglos/C#/Insercion al final .cs
using System;
class program
{
    static void Main(string [] args)
    {
        int [] arr = {11, 21, 31, 41, 51, 61};
        
        //El arreglo antes de la inserción
        Console.WriteLine("Antes de la inserción el array es: ");

        foreach (int elemento in arr)
        {
            Console.Write(elemento + " ");
        }
        Console.WriteLine();

        
        int ele = 52;
        
        //Hacemos un nuevo array
        int [] nuevoArr = new int[arr.Length + 1];
        

        for (int i = 0; i < arr.Length; i++)
        {
            nuevoArr[i] = arr[i];
        }
        //Se mueve a la ultima posicion
        nuevoArr[arr.Length] = ele;

        //El arreglo despues de la inserción
        Console.WriteLine("El array despues de la inserción es: ");
        foreach (int elemento in nuevoArr)
        {
            Console.Write(elemento + " ");
        }
    }
}